docs/details/dashboards/dashboard-home.md

```diff
@@ -33,3 +25,44 @@ This section lists all added hosts along with essential information about their
 * RAM
 * Host Uptime
 * DB Uptime
+* Advisors check
+
+ This panel also displays the current version number. Use **Upgrade to X.X.X version** to upgrade to the most recent version of PMM.
+
+
+## Anomaly Detection
+
+The **Anomaly Detection** panel lists all the anomalies in your environment. Color-coded states on the panels provide a quick visual representation of the problem areas.
+
+The following anomalies are displayed on this panel:
+
+* CPU anomalies (high as well as low)
+* High CPU servers
+* Low CPU servers
+* Disk Queue anomalies
+* High disk queue
+* High Memory Used
+
+
+## Command Center
+
+You can find critical information such as CPU utlization, memory utilization, anomalies, read and write latency, etc., about your environment on the **Command Center** panel. This information is represented graphically. In this panel, the graphs for the last hour and the previous week are displayed adjacently, making it easy to identify the trends.
```

Comment:
Can we highlight that command center graphs list the Top 20 ? @rasika-chivate

Reply:
@dygksy Could you please elaborate on this...